go.mod

```diff
@@ -1,3 +1,12 @@
 module github.com/emicklei/go-restful/v3
 
 go 1.13
+
```

Comment:
These dependencies should not be required. Most likely they are added because of the examples. I will revisit this.

Reply:
please remove the go.mod and go.sum from the PR

Reply:
backed those out with: e2c750eb4b5f3a84e37508c7919eb9ed847b794c